test/module/irohad/torii/torii_queries_test.cpp

```diff
@@ -84,22 +77,330 @@ class ToriiServiceTest : public testing::Test {
 
   ServerRunner *runner;
   std::thread th;
+
+  PCSMock pcsMock;
+  StatelessValidatorMock statelessValidatorMock;
+  WsvQueryMock wsv_query;
+  BlockQueryMock block_query;
 };
 
+/**
+ * Test for error response
+ */
+
+TEST_F(ToriiServiceTest, FindWhenResponseInvalid) {
+  EXPECT_CALL(statelessValidatorMock,
```

Comment:
This method should be called once in this test. Please use `.Times(1)` or `.WillOnce()`